soufi/finders/centos.py

```diff
@@ -54,14 +54,8 @@ def __init__(
     ):
         if not repos or optimal_repos is True:
             repos = DEFAULT_SEARCH
-        if not source_repos:
-            source_repos = self.get_source_repos(repos, optimal_repos)
-            self._reinit_source_for_walk = True
-            self._reinit_args = (repos, optimal_repos)
-        if not binary_repos:
-            binary_repos = self.get_binary_repos(repos, optimal_repos)
-            self._reinit_binary_for_walk = True
-            self._reinit_args = (repos, optimal_repos)
+        self.repos = repos
+        self.optimal_repos = optimal_repos
```

Comment:
I feel so silly for not thinking of this.  It's really obvious in hindsight.

Because GitHub is awesome and disallows commenting on LoC not explicitly under review, the private attributes on L43-44 are now unused and can be removed.

Reply:
Ah woopsie!